src/pages/TxnView.js

```diff
@@ -311,6 +374,13 @@ class TxnView extends Component {
                   />
                   <a href={'/blocks/' + txn.height}>{txn.height}</a>
                 </p>
+                {txn.type === 'rewards_v1' && (
+                  <p style={{ color: '#FFC769' }}>
+                    <WalletOutlined style={{ color: '#FFC769', marginRight: 6, }}/>
+                    { txn.totalAmount.floatBalance }
```

Comment:
rather than using `floatBalance` here, I'd suggest using the `toString` method which accepts a max decimal places and appends the currency type ticker.

```suggestion
                    { txn.totalAmount.toString(2) }
```

Reply:
should I attach toString() to the `totalAmount` object or floatBalance?

Reply:
Hii @allenan. I got it. Definitely, it looks much better with currency type. Thank You for your review.

Reply:
I included a suggestion above, just test it to make sure it does what you want it to

Reply:
Sorry I couldn't get it when I first saw it. I got confused about the `toString()` method.  I learned a new thing today.  Thank you for giving the suggestion,